services/blockchain-indexer/app.js

```diff
@@ -86,6 +86,10 @@ initDatabase()
 			await tempApp.run();
 			const { getRegisteredModules } = require('./shared/constants');
 			registeredModules.push(...await getRegisteredModules());
+
+			// TODO: remove after getSystemMetadata returns NFT
+			registeredModules.push('nft');
```

Comment:
could it create any unexpected behaviour when `getSystemMetadata` starts returning `nft`? would it be better to check and add only when `nft` is not present?